scripts/plh-spreadsheet/plh-spreadsheet.model.ts

```diff
@@ -1,10 +1,17 @@
 import { RapidProFlowExport } from 'src/app/shared/services/chat/offline/rapid-pro-export.model';
 
 export interface ContentIndexRow {
-    Sheet_Name: string,
-    Content_Type: "Conversation" | "Toolbox",
+    //Removed these as adapting to the format used by Ohad.
+    //Sheet_Name: string,
+    //Content_Type: "Conversation" | "Toolbox",
+    Flow_Type: "Conversation" | "Toolbox",
+    Module?: string,
+    Flow_Name: string,
     Character?: "Friend" | "Guide",
-    Entry_Condition?: string
+    Second_Character?: string,
+    Entry_Condition?: string,
+    Output?: string,
+    Comment_Suggestion?: string
```

Comment:
@michaelclapham we should probably discuss these changes to make sure I don't break your existing code. I adapted this structure based on Excel files provided, but I guess there are different versions out there. So if we can decide on a structure then we can be consistent.